docs/user_guide/SIMP_Administration/Classification_and_Data/SIMP_Compliance_Engine.inc

```diff
@@ -0,0 +1,112 @@
+.. _SIMP Compliance Engine:
+
+The SIMP Compliance Engine
+--------------------------
+
+The :term:`SIMP Compliance Engine` allows users to enable both Puppet
+parameter-level enforcement and validation of compliance with regulatory and
+internal policies.
+
+This capability is provided by the `simp-compliance_markup`_ puppet module and
+will be renamed at some point in the future to `simp-compliance_engine` for
+clarity.
+
+.. IMPORTANT::
+
+   Your system must be configured to use at least :term:`Hiera` version 5 for
+   the Compliance Engine to function properly.
+
+   SIMP 6.3+ supports this capability out of the box bue earlier versions may
+   need to `perform a Hiera migration`_.
+
+Enabling the SIMP Compliance Engine
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the Puppet implementation, the Compliance Engine is a Hiera backend that
+provides lookup capabilities into specially formatted :term:`SIMP compliance
+profile` data sets.
+
```

Comment:
May be useful to add a sentence about how the lookup capabilities are used.

Reply:
Linked over to APL, etc... At some point, you're not going to get far with working with it if you don't understand at least a bit of where the data is coming from.

Reply:
I know this is closed but...
I think you should add (not sure if its already present) a list of "minimal Puppet knowledge" required to operate SIMP to the beginning of the documentation.